pkg/sdn/apis/network/v1/types.go

```diff
@@ -28,6 +30,12 @@ type ClusterNetwork struct {
 	PluginName string `json:"pluginName,omitempty" protobuf:"bytes,5,opt,name=pluginName"`
 }
 
+// PLACEHOLDER
```

Comment:
As of now the system fills each subnet from first entry (top) to last, in the future there are plans to label the subnets and label the nodes and allocate based on that label but that does not exist in this PR.